src/main/webapp/i18n/de/quizExercise.json

```diff
@@ -213,7 +215,12 @@
             },
             "explanationAnswered": "Du hast die Frage beantwortet",
             "explanationNotAnswered": "Du hast die Frage noch nicht beantwortet",
-            "submissionWarning": "Bist du sicher, dass du einreichen willst? Du hast nicht alle Fragen beantwortet und hast noch etwas Zeit übrig!"
+            "submissionWarning": "Bist du sicher, dass du einreichen willst? Du hast nicht alle Fragen beantwortet und hast noch etwas Zeit übrig!",
+            "placeholder": {
+                "title": "Kurzer Quiz Titel",
+                "questionTitle": "Frage Titel",
+                "shortQuestionTitle": "Kurzer Frage Title"
```

Comment:
```suggestion
                "questionTitle": "Fragen Titel",
                "shortQuestionTitle": "Kurzer Fragen Title"
```

Reply:
Applying your suggestion, shouldnt it then be "Kurzer Fragen Titel" or even "kurzer Fragentitel" (Fragen-Titel)?

Reply:
Yes, maybe the English version can be changed as well. I actually did not change the existing ones, I just moved them to translation files. I've tried to add missing German translations but  my German is not so good.

> Applying your suggestion, shouldnt it then be "Kurzer Fragen Titel" or even "kurzer Fragentitel" (Fragen-Titel)? (It's still Title - should at least be Titel)

I committed the previous suggestion but should I change it again?